server/datastore/mysql/migrations/tables/20240110134315_AddDisplayVersionToOSTable.go

```diff
@@ -0,0 +1,42 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240110134315, Down_20240110134315)
+}
+
+func Up_20240110134315(tx *sql.Tx) error {
+	addColumnStmt := `
+		ALTER TABLE operating_systems
+		ADD COLUMN display_version VARCHAR(10) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '';
```

Comment:
I like the simplicity of using a zero value, over using a NULL column + a pointer struct field.  Curious if there are strong opinions here.